moto/redshiftdata/urls.py

```diff
@@ -0,0 +1,10 @@
+from .responses import RedshiftDataAPIServiceResponse
+
+url_bases = [
+    r"https?://redshift-data\.(.+)\.amazonaws\.com",
```

Comment:
Whenever the URLs change, you should also call `scripts/update_backend_index.py`.

There is a hardcoded index of all these URL's to speed things up, and ensure Moto doesn't have to load every module to get the list of URL's - this script will automatically update this index.

Reply:
That's good to know! Thank you